ports/sonic.1/Sonic 1.sh

```diff
@@ -13,28 +13,85 @@ else
 fi
 
 source $controlfolder/control.txt
-source $controlfolder/device_info.txt
-
-[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
-
 get_controls
 
-$ESUDO chmod 666 /dev/tty1
+# Source Device Info
+source $controlfolder/device_info.txt
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
 
+# Set variables
 GAMEDIR="/$directory/ports/sonic1"
 > "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
 
+# Set current virtual screen
+if [ "$CFW_NAME" == "muOS" ]; then
+  /opt/muos/extra/muxlog & CUR_TTY="/tmp/muxlog_info"
+elif [ "$CFW_NAME" == "TrimUI" ]; then
+  CUR_TTY="/dev/fd/1"
+else
+  CUR_TTY="/dev/tty0"
+fi
+
 cd $GAMEDIR
 
-export LD_LIBRARY_PATH="$GAMEDIR/libs:$LD_LIBRARY_PATH"
-export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
+# Exports
+LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$GAMEDIR/libs"
```

Comment:
What's wrong with our standard?

We do it via export so in case child processed need it its still available.
We do :$LD_LIBRARY_PATH" at the end so system libs will be used first. 
Our default template for ports we have in here https://portmaster.games/packaging.html#the-launchscript-sh
If we need to change something we need to update the template and apply it to future ports.